andrewinsoul/Fast-Food-ClientSide
# #162028816 Add landing page and links to login and signup components

#### What does this PR do?
This PR creates the landing page of the application with React and creates link to other pages

#### Description of Tasks completed
- Create landing page component
- Create components for signup and login
- Create links between all created components

#### How should this be manually tested
```
- Clone Repo
- Run npm install to install all dependencies
- Run npm start
- Your Browser should open with the favicon showing and a view of the landing page with links to signup and login page
```
#### Any Background context provided?
- N/A

#### What are the relevant Pivotal Tracker Stories?
[#162028816](https://www.pivotaltracker.com/story/show/162028816)
